README.md

```diff
@@ -29,6 +29,12 @@ This sample demonstrates a .NET 4.6.1 Web App (MVC) application calling a ASP.NE
 
 > Looking for previous versions of this code sample? Check out the tags on the [releases](../../releases) GitHub page.
 
+> There's a newer version of this sample! Check it out: <https://github.com/azure-samples/ms-identity-aspnet-daemon-webapp>

```

Comment:
I think this should be just after the build badge.
Let's discuss tomorrow, @kjyam98

Reply:
@jmprieur  I moved the v2 upsell message to just below the build badge - please review the PR.